### PR TITLE
WD-30: Sort order

### PR DESCRIPTION
**Pull request by cakoshakib**
## Changes

- Added task_order table to maintain sort order of each day
- `/order` endpoint
- Modified GET `/tasks` responses
- POST `/task` and DELETE `/task` update sort orders
- Created POST `/botauth` for bot authorization

### CODE REVIEW

File: api/auth.go
Comment (markdstouffer) [inline api/auth.go:1]: Nice work, but WHY is this in this branch lol..

File: main.go [hunk 1]
```diff
@@ -19,22 +20,25 @@ func load_env() {
...
 
 func main() {
+	dev := flag.Bool("dev", false, "")
+	flag.Parse()
+	if *dev {
+		load_env()
+		router := handler.Router()
+		router.Run()
+		return
+	}
```
Comment (markdstouffer) [inline main.go:36]: :fire:

**Review by markdstouffer — APPROVED** [review]
LGTM!

File: api/auth.go
Comment (cakoshakib) [reply api/auth.go:1]: :3

**Pull request merged**